CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # 2.0.0
 
-* Bumped Ion Go to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
+* Bumped Ion Go dependency to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
 * Logging interface now provides the LogLevel as a parameter.
 * Expose transaction ID in Transaction interface.
 
+## :boom: Breaking changes
+
+* The `Logger` interface's `Log` method now takes in a `LogLevel` to specify the logging verbosity. Any instances of `Logger.Log()` will need to be updated accordingly.
```

Comment:
Can we add code example? on how users might use these?

Reply:
Done.

Reply:
Can we indent the code snippets so they are inline with the bullet points?

Reply:
Done.